src/eiffel_graphql_api/storage.py

```diff
@@ -30,39 +34,76 @@
         os.environ["RABBITMQ_USERNAME"] = username.read()
 
 
-def insert_to_db(event, context):
-    print(event)
-    collection = DATABASE[event.meta.type]
-    collection.create_index([("meta.id", pymongo.ASCENDING)])
-    collection.create_index([("links.target", pymongo.ASCENDING)])
-    doc = event.json
-    doc['_id'] = event.meta.event_id
-    try:
-        collection.insert_one(doc)
-    except pymongo.errors.DuplicateKeyError:
-        print("Event already exists in the database, "
-              "skipping: {}".format(event.json))
-        return False
-    return True
+def parse_args(args):
+    """Parse command line parameters
 
+    :param args: Commandl ine parameters as list of strings.
+    :type args: list
+    :return: Command line parameters namespace.
+    :rtype: :obj:`argparse.Namespace
+    """
+    parser = argparse.ArgumentParser(
+        description="Tool for storing eiffel events in a Mongo database.")
+    parser.add_argument(
+        "--version",
+        action="version",
+        version="eiffel-graphql-storage {ver}".format(ver=__version__))
+    parser.add_argument(
+        "-v",
+        "--verbose",
+        dest="loglevel",
+        help="set loglevel to DEBUG",
+        action="store_const",
+        default=logging.INFO,
+        const=logging.DEBUG)
+    return parser.parse_args(args)
 
-if __name__ == "__main__":
-    DATABASE = get_database()
-    SSL = os.getenv("RABBITMQ_SSL", "false") == "true"
-    DATA = {
+
+def setup_logging(loglevel):
+    """Setup basic logging
+
+    :param loglevel: Minimum loglevel for emitting messages.
+    :type loglevel: int
+    """
+    logformat = "[%(asctime)s] %(levelname)s:%(message)s"
+    logging.basicConfig(level=loglevel, stream=sys.stdout,
+                        format=logformat, datefmt="%Y-%m-%d %H:%M:%S")
```

Comment:
Excellent! When I get around to #9 I'll add an argument for providing a logging config file, but this will stay as a baseline for those who are less picky about their logging.